src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -90,11 +92,13 @@ private static String pluralize(int count, String singular, String plural) {
 		return count == 1 ? singular : plural;
 	}
 
-	private static String nullSafeMessage(AssertionError failure) {
+	private static String nullSafeMessage(Throwable failure) {
+		if (failure == null) {
```

Comment:
I'm with @kcooney: calling `addFailure(null)` isn't a supported use case. So, I think we should move the check there and throw an `IllegalArgumentException` when `failure` is `null`.